### ISSUE TITLE
MarkerWithLabel not working react typescript

### ISSUE THREAD

**Issue by adithyamaheshb** [mxl/react-google-maps-example#2]
`import MarkerWithLabel from "react-google-maps/lib/components/addons/MarkerWithLabel";
import getRandomInt from './util';

const someLatLng = { lat: 55.751244, lng: 37.618423 }

const location = {
    lat: 39.0997,
    lng: -94.5786
};

    const MyGoogleMap = withScriptjs(withGoogleMap(() =>
        <GoogleMap
            defaultCenter={ location }
            defaultZoom={4}
            //options={{ disableDefaultUI: true }}
        >
            <MarkerWithLabel position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }}
                defaultAnimation={2}
                //markerWithLabel={MarkerWithLabel}
                labelAnchor={new google.maps.Point(35, 27)} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />
            <Marker position={{ lat: getRandomInt(35.8565, 40.5865), lng: getRandomInt(-122.5896, -76.9672) }} defaultAnimation={2} />

        </GoogleMap>))
    const loadingElement = <div />
    const containerElement = <div style={{ height: '100vh' }} />
    const mapElement = <div style={{ height: '100vh' }} />

export default class Map2 extends React.Component<any, any> {
       constructor(props: any) {
        super(props);
     }

    render() {
        return (
            <MyGoogleMap loadingElement={loadingElement}
                containerElement={containerElement}
                googleMapURL={googleMapURL}
                mapElement={mapElement} />
            )
    }
}`

And I get the following Error

Uncaught Error: React.Children.only expected to receive a single React element child.
    at invariant (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:118)
    at Object.onlyChild [as only] (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:49413)
    at MarkerWithLabel.componentDidMount (MarkerWithLabel.js:124)
    at vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:39707
    at measureLifeCyclePerf (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:39518)
    at vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:39706
    at CallbackQueue.notifyAll (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:22435)
    at ReactReconcileTransaction.close (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:44602)
    at ReactReconcileTransaction.closeAll (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:17176)
    at ReactReconcileTransaction.perform (vendor.js?v=kmpFKjrR27bbuRgrzphkt1jtJWQAZ9qKENOVBubjIYU:17123)

and

Uncaught TypeError: Cannot read property 'nodeType' of undefined
    at MarkerLabel_.setContent (index.js:355)
    at MarkerLabel_.draw (index.js:343)
    at Qu.draw (overlay.js:3)
    at Ru.Jb (overlay.js:4)
    at eu (VM2199 map.js:38)
    at VM2199 map.js:41

**Comment by rgllm** [comment]
+1

**Comment by howardesther13** [comment]
Did you guys find the solution? I am facing the same issue

**Comment by rgllm** [comment]
Yes @howardesther13, I found a solution. What error are you getting?

**Comment by StefanoSega** [comment]
I'm using react-google-maps v9.4.5 and is not working as well.

the code looks like:
```
const MyMapComponent = compose(
      withProps({
        googleMapURL:
          "https://maps.googleapis.com/maps/api/js?key=...&v=3.exp&libraries=geometry,drawing,places",
        loadingElement: <div style={{ height: `100%` }} />,
        containerElement: <div style={{ height: `400px` }} />,
        mapElement: <div style={{ height: `100%` }} />
      }),
      withScriptjs,
      withGoogleMap
    )(() => (
        <GoogleMap defaultZoom={8} defaultCenter={{ lat: -34.397, lng: 150.644 }}>
<MarkerWithLabel
    position={{
      lat: -34.397, lng: 150.644 
    }}
    labelAnchor={new google.maps.Point(0, 0)}
    labelStyle={{backgroundColor: "yellow", fontSize: "32px", padding: "16px"}}
  >
    <div>Hello There!</div>
  </MarkerWithLabel>
        </GoogleMap>
      )
    );
```

I get this error on the browser console:
```
index.js:355 Uncaught TypeError: Cannot read property 'nodeType' of undefined
    at MarkerLabel_.setContent (index.js:355)
    at MarkerLabel_.draw (index.js:343)
    at Fx.draw (overlay.js:4)
    at Gx.Ob (overlay.js:4)
    at Zw (map.js:39)
    at map.js:40
```

**Comment by ghulam-dev** [comment]
getting same error
